calico/getting-started/kubernetes/quickstart.md

```diff
@@ -10,20 +10,24 @@ This quickstart gets you a single-host Kubernetes cluster with {{site.prodname}}
 
 ### Value
 
-Use this quickstart to quickly and easily try {{site.prodname}} features. To deploy a cluster suitable for production, refer to [{{site.prodname}} on Kubernetes]({{ site.baseurl }}/getting-started/kubernetes/).
+Use this quickstart to quickly and easily try {{site.prodname}} features. To deploy a cluster suitable for production, refer to [{{site.prodname}} on Kubernetes]({{site.baseurl}}/getting-started/kubernetes/).
 
 ### Before you begin
 
-- Make sure you have a linux host that meets the following requirements:
+**Required**
+
+- A Kubernetes cluster with a [supported version](({{site.baseurl}}/getting-started/kubernetes/requirements)
```

Comment:
This isn't true - the quickstart has the user create a cluster, so it's not required that a user has a pre-existing cluster at the start of the doc.